assets/styles/main.scss

```diff
@@ -782,6 +782,9 @@ body.sticky-offset {
 }
```

Comment:
Now that one field/row occupies multiple lines, you need to increase the spacing between the rows. Otherwise, it's not obvious which input each label is for.

Reply:
How could exclude the nested tables from this? :D

Reply:
Otherwise everything would be spaced greater, which wouldn't benefit anyone.

Reply:
`tr:not(.item)` should work.

Reply:
Maybe you need to extend the `:not()`. I haven't checked whether there are any other nested tables.